src/Config/RuntimeConfigLoader.cs

```diff
@@ -52,10 +55,16 @@ public static bool TryParseConfig(string json, [NotNullWhen(true)] out RuntimeCo
                 return false;
             }
 
+            string updatedConnectionString = config.DataSource.ConnectionString;
+
             if (!string.IsNullOrEmpty(connectionString))
```

Comment:
Would this be true for hosted scenario?

Reply:
Yes, TryParseConfig is called in Both Scenario, where we update the conn_string.